package.json

```diff
@@ -36,12 +36,15 @@
     "expose-loader": "~0.6.0",
     "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.9.0",
+    "highlight.js": "^9.12.0",
```

Comment:
if those depenecides are used only for storybook, we should add them to the build ignore white list, similar to https://github.com/theforeman/foreman-packaging/pull/2846

Reply:
Done, see https://github.com/theforeman/foreman-packaging/pull/2851 and https://github.com/theforeman/foreman-packaging/pull/2852